build_tools/docker/base/fetch_cuda_deps.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# While IREE can be built with a normal CUDA Toolkit installation, the compiler
+# and runtime only actually depend on a very thin (platform independent) slice
```

Comment:
They depend on it to *build*. Doesn't the runtime need the full thing to actually run?